src/constants/api.jsx

```diff
@@ -1,4 +1,4 @@
-const API_LINK: string = "https://api.fda.gov"
+const API_LINK: string = "https://api.openfda-ndc-demo.semanticbits.com"
```

Comment:
@beardedfinch may need to be updated

Reply:
Fixed this and a couple other deploy issues.